contracts/test/integration/vip/vip15.sol

```diff
@@ -0,0 +1,477 @@
+                arguments: abi.encodeWithSignature(
+                    "withdrawAllERC20ToSafeAddress(address,address)",
+                    MainnetAddresses.VOLT_USDC_PSM,
+                    MainnetAddresses.VOLT
+                ),
+                description: "Withdraw all VOLT from USDC PSM"
+            })
+        );
+
+        /// ------- pause morpho deposits -------
+
+        proposal.push(
+            ITimelockSimulation.action({
+                value: 0,
+                target: MainnetAddresses.MORPHO_COMPOUND_USDC_PCV_DEPOSIT,
+                arguments: abi.encodeWithSignature("pause()"),
+                description: "Pause USDC Morpho Compound PCV Deposit"
+            })
+        );
+
+        proposal.push(
+            ITimelockSimulation.action({
+                value: 0,
+                target: MainnetAddresses.MORPHO_COMPOUND_DAI_PCV_DEPOSIT,
+                arguments: abi.encodeWithSignature("pause()"),
+                description: "Pause USDC Morpho Compound PCV Deposit"
+            })
+        );
+
+        /// ------- pause psms -------
+
+        proposal.push(
+            ITimelockSimulation.action({
+                value: 0,
+                target: MainnetAddresses.VOLT_USDC_PSM,
+                arguments: abi.encodeWithSignature("pause()"),
+                description: "Pause USDC PSM"
+            })
+        );
+
+        proposal.push(
+            ITimelockSimulation.action({
+                value: 0,
+                target: MainnetAddresses.VOLT_DAI_PSM,
+                arguments: abi.encodeWithSignature("pause()"),
+                description: "Pause DAI Morpho Compound PCV Deposit"
+            })
+        );
+
+        /// ------- pause allocator -------
+
+        proposal.push(
+            ITimelockSimulation.action({
+                value: 0,
+                target: MainnetAddresses.ERC20ALLOCATOR,
+                arguments: abi.encodeWithSignature("pause()"),
+                description: "Pause ERC20 Allocator"
+            })
+        );
+
+        /// ------- role revoked in core -------
+
+        proposal.push(
+            ITimelockSimulation.action({
+                value: 0,
+                target: MainnetAddresses.CORE,
+                arguments: abi.encodeWithSignature(
+                    "revokePCVController(address)",
+                    MainnetAddresses.MORPHO_COMPOUND_PCV_ROUTER
+                ),
+                description: "Revoke PCV Controller from Morpho Compound PCV Router"
+            })
+        );
+
+        proposal.push(
+            ITimelockSimulation.action({
+                value: 0,
+                target: MainnetAddresses.CORE,
+                arguments: abi.encodeWithSignature(
+                    "revokePCVController(address)",
+                    MainnetAddresses.ERC20ALLOCATOR
+                ),
+                description: "Revoke PCV Controller from ERC20 Allocator"
+            })
+        );
```

Comment:
```suggestion
        );

        proposal.push(
            ITimelockSimulation.action({
                value: 0,
                target: MainnetAddresses.CORE,
                arguments: abi.encodeWithSignature(
                    "revokePCVController(address)",
                    MainnetAddresses.PCV_GUARDIAN
                ),
                description: "Revoke PCV Controller from PCV_GUARDIAN"
            })
        );

        proposal.push(
            ITimelockSimulation.action({
                value: 0,
                target: MainnetAddresses.CORE,
                arguments: abi.encodeWithSignature(
                    "revokePCVController(address)",
                    MainnetAddresses.GOVERNOR
                ),
                description: "Revoke PCV Controller from Multisig"
            })
        );
```

Also revoke PCV_CONTOLLER from PCVGuardian & Multisig

Reply:
this change has been integrated into the PR